ProjectManager/ProjectManager/App/Source/Model/TaskManager.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  TaskManager.swift
+//  ProjectManager
+//
+//  Created by Harry, KokkiLE on 2023/05/22.
+//
+
+import Foundation
+import Combine
+
+final class TaskManager {
+    static let shared = TaskManager()
+    
+    @Published private var taskList: [Task] = []
+    
+    private init() { }
+    
+    func taskListPublisher() -> AnyPublisher<[Task], Never> {
+        return $taskList.eraseToAnyPublisher()
+    }
+    
+    func create(task: Task) {
+        taskList.append(task)
+    }
+    
+    func update(task: Task) {
+        let index = taskList.firstIndex { $0.id == task.id }
+        guard let index else { return }
```

Comment:
```suggestion
       guard let index = taskList.firstIndex(where: { $0.id == task.id }) else {
            return
      }
```
이렇게도 할 수 있을 것 같은데, 취향의 차이가 있을 것 같네요 :)

Reply:
오 넵! 저희도 한 줄로 적는게 더 좋은 것 같아서 수정했습니다 🫡